cmd/run.go

```diff
@@ -65,6 +66,10 @@ var runCmd = &cobra.Command{
 		go func() {
 			err := proj.RunServices(localCloud, stopChan, updatesChan)
 			if err != nil {
+				if strings.Contains(err.Error(), "Docker daemon is not running") {
+					tui.CheckErr(err)
+				}
```

Comment:
we might be able to check this at the start instead. Or use the command dependencies to ensure everything is running?